spec/yearmonth.html

```diff
@@ -3,5 +3,511 @@
+      <h1>RegulateYearMonth ( _year_, _month_, _disambiguation_ )</h1>
+      <emu-alg>
+        1. Assert: _disambiguation_ is one of `"constrain"`, `"balance"`, or `"reject"`.
+        1. If _disambiguation_ is `"constrain"`, then
+          1. <mark>TODO: define.</mark>
+        1. If _disambiguation_ is `"balance"`, then
+          1. Return BalanceYearMonth(_year_, _month_).
+        1. If _disambiguation_ is `"reject"`, then
+          1. If ! ValidateYearMonth(_year_, _month_) is *false*, then
+            1. Throw a *RangeError* exception.
+          1. Return the new Record {
+            [[Year]]: _year_,
+            [[Month]]: _month_,
+            }.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id=sec-temporal-ValidateYearMonth aoid=ValidateYearMonth>
+      <h1>ValidateYearMonth ( _year_, _month_ )</h1>
+      <emu-alg>
+        1. Assert: _year_ and _month_ are integer Number values.
+        1. If _month_ &lt; 1 or _month_ &gt; 12, then
+          1. Return *false*.
+        1. Return *true*.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id=sec-temporal-BalanceYearMonth aoid=BalanceYearMonth>
+      <h1>BalanceYearMonth ( _year_, _month_ )</h1>
+      <emu-alg>
+        1. Assert: _year_ and _month_ are integer Number values.
+        1. Set _year_ to _year_ + floor((_month_ - 1) / 12).
+        1. Set _month_ to NonNegativeModulo(_month_ - 1, 12) + 1.
+        1. Return the new Record {
+          [[Year]]: _year_,
+          [[Month]]: _month_,
+          }.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id=sec-temporal-createyearmonth aoid=CreateYearMonth>
+      <h1>CreateYearMonth ( _year_, _month_ [ , _newTarget_ ] )</h1>
+      <emu-alg>
+        1. Assert: ! ValidateYearMonth(_year_, _month_) is *true*.
+        1. If _newTarget_ is not given, set it to %Temporal.YearMonth%
+        1. Let _object_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%Temporal.YearMonth.prototype%"`, « [[InitializedTemporalYearMonth]], [[Year]], [[Month]] »).
+        1. Set _object_.[[Year]] to _year_.
+        1. Set _object_.[[Month]] to _month_.
+        1. Return _object_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-ParseYearMonthString" aoid="ParseYearMonthString">
+      <h1>ParseYearMonthString ( _yearMonth_ )</h1>
+      <emu-alg>
+        1. Assert: Type(_yearMonth_) is String.
+        1. Let _valid_ be *true* if _yearMonth_ matches <a href="https://github.com/tc39/proposal-temporal/blob/main/polyfill/lib/regex.mjs">the "yearmonth" regular expression in the polyfill</a> and *false* otherwise.
+        1. <mark>TODO: Actually specify this somehow.</mark>
+        1. If _valid_ is *false*, then
+          1. Throw a *RangeError* exception.
+        1. Let _year_ and _month_ be the respective parts of _yearMonth_ interpreted as decimal numbers.
+        1. Return the new Record {
+          [[Year]]: _year_,
+          [[Month]]: _month_,
```

Comment:
all these multiline records seem indented incorrectly; the slots should get an additional level